sdk/loadtestservice/Azure.ResourceManager.LoadTesting/src/autorest.md

```diff
@@ -26,17 +26,18 @@ rename-mapping:
   LoadTestResource: LoadTestingResource
   QuotaResource: LoadTestingQuota
   CheckQuotaAvailabilityResponse: LoadTestingQuotaAvailabilityResult
-  EncryptionProperties: CustomerManagedKeyEncryptionProperties
-  EncryptionPropertiesIdentity: CustomerManagedKeyIdentity
+  EncryptionProperties: LoadTestingCmkEncryptionProperties
+  EncryptionPropertiesIdentity: LoadTestingCmkIdentity
   EndpointDependency: LoadTestingEndpointDependency
   EndpointDetail: LoadTestingEndpointDetail
   OutboundEnvironmentEndpointCollection: OutboundEnvironmentEndpointListResult
```

Comment:
This is the only think now that does not have LoadTesting prefix?

Reply:
This model is internal, so we can just leave it as it is.